src/persistentReducedStream.ts

```diff
@@ -48,9 +49,31 @@ export const persistentReducedStream = <State>(
     tag(name)
   );
 
-  const stream = new ObservableState(name, source$, initialState);
+  try {
```

Comment:
Why are you using try-catch instead of module.hot?

Reply:
Good question!

This file is not the file that is being hot reloaded, it is the file that *uses* this function that will be reloaded. So that file will run multiple times and register a persistentReducedStream each time, which typically is a duplicate reducer error that needs to be handled in order to allow HMR to happen.